cli/task_runner.rs

```diff
@@ -271,7 +271,6 @@ impl ShellCommand for NpmPackageBinCommand {
   ) -> LocalBoxFuture<'static, ExecuteResult> {
     let mut args = vec![
       "run".to_string(),
-      "-A".to_string(),
```

Comment:
This won't work because binaries like `vite` (which this is emulating the launch of) are not secure by default and do not have the concept of permission flags. If someone wants to have permissions, they can switch to run with `deno` instead.